tests/runner/Cargo.toml

```diff
@@ -9,10 +9,16 @@ edition = "2021"
 [features]
 default = ["bios", "uefi"]
 bios = ["bootloader/bios"]
-uefi = ["bootloader/uefi", "dep:ovmf-prebuilt"]
+uefi = ["bootloader/uefi"]
 
 [dependencies]
 bootloader = { path = "../..", default-features = false }
 strip-ansi-escapes = "0.1.1"
-ovmf-prebuilt = { version = "0.1.0-alpha.1", optional = true }
 wait-timeout = "0.2.0"
+anyhow = "1.0.89"
+ureq = "2.10.1"
+fs-err = "2.11.0"
```

Comment:
It seems easy to avoid this extra dependency by writing out the error handling code yourself. It is only used in a couple of places anyway.

Reply:
Your right fs-err and anyhow are not really necessary here.